vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_wNa16.py

```diff
@@ -119,14 +127,21 @@ def create_weights(self, layer: torch.nn.Module, input_size: int,
                                                           dtype=torch.int64),
                                          weight_loader=weight_loader)
 
+        # G_IDX (for activation reordering)
+        g_idx = BasevLLMParameter(data=torch.empty(input_size_per_partition,
+                                                   dtype=torch.int32),
+                                  weight_loader=weight_loader)
```

Comment:
Is it okay to make this parameter in every case? What about older checkpoints that don't have this parameter?

Reply:
I think my question was worded weirdly, sorry. I am just concerned about the weight loader trying to find this parameter in the checkpoint, and it not being present.

Reply:
I regression tested using `neuralmagic/TinyLlama-1.1B-Chat-v1.0-marlin` without issue

Reply:
I'd just update to only create the parameter if `actorder` is True

Reply:
This is because the g_idx passed to the kernel is conditional on the `actorder` flag in the config
https://github.com/vllm-project/vllm/pull/6358/files#diff-df5f822218e5ac1430f35a806bc9cebd78c99cfe1e6738de89ae3e9f5a1fdbecR162

Reply:
If self.actorder is True, it'll use the created parameter. Otherwise, it'll create an empty one. So I dont think you need to initialize it here if `self.actorder` is False

Reply:
Yeah that seems to be the case from this else-case later - so no need to make the parameter
```python
            layer.weight_g_idx = marlin_make_empty_g_idx(device)
            layer.g_idx_sort_indices = marlin_make_empty_g_idx(device)
```